setup.py

```diff
@@ -4,37 +4,52 @@
 import os
 import setuptools
 
-__version__ = '0.0.1'
+
+__version__ = '0.0.4'
 CPATH = os.getenv("CPATH")
-print("CPATH:")
+LD_LIBRARY_PATH = os.getenv("LD_LIBRARY_PATH")
+# Strip any colons
+CPATH = CPATH.strip(":")
+LD_LIBRARY_PATH = LD_LIBRARY_PATH.strip(":")
+print("CPATH")
 print(CPATH)
+print("LD_LIBRARY_PATH")
+print(LD_LIBRARY_PATH)
 
 
 class get_pybind_include(object):
-    """Helper class to determine the pybind11 include path
+  """Helper class to determine the pybind11 include path
 
-    The purpose of this class is to postpone importing pybind11
-    until it is actually installed, so that the ``get_include()``
-    method can be invoked. """
+  The purpose of this class is to postpone importing pybind11
+  until it is actually installed, so that the ``get_include()``
+  method can be invoked. """
 
-    def __init__(self, user=False):
-        self.user = user
+  def __init__(self, user=False):
```

Comment:
I dislike mismatched types when doing flags like this. `self.user` will likely be some object whenever it is eventually assigned yes? (or is it a string or some other primitive?). Thus having either an object (has been assigned) or a bool (unassigned) mismatch.

if its an object eventually use `None` as the falsy.

Reply:
Good comment. I copy/pasted this code from a PyBind11 tutorial, but we should clean this up a bit. I'll add this onto the clean-up list for the repo.